src/compnonents/home/RankingsTable.svelte

```diff
@@ -7,38 +7,63 @@
 		TableHead,
 		TableHeadCell
 	} from 'flowbite-svelte';
+	import type { Ranking } from 'src/types/Ranking';
+
+	export let worldRankingsFemale: Ranking[];
+	export let worldRankingsMale: Ranking[];
+	export let male: boolean = true;
+
+	function capitalize(word: string): string {
+		return word.charAt(0).toUpperCase() + word.slice(1);
+	}
 </script>
 
 <Table hoverable={true}>
 	<TableHead class="text-black/25">
 		<TableHeadCell>Rank</TableHeadCell>
 		<TableHeadCell>Full Name</TableHeadCell>
 		<TableHeadCell>Country</TableHeadCell>
-		<TableHeadCell>Points</TableHeadCell>
-		<TableHeadCell>Pers. Best</TableHeadCell>
+		<TableHeadCell>Date</TableHeadCell>
+		<TableHeadCell>Time</TableHeadCell>
 	</TableHead>
 
 	<TableBody>
-		<TableBodyRow class="border-t-[1px]">
-			<TableBodyCell>1</TableBodyCell>
-			<TableBodyCell>Veddriq LEONARDO</TableBodyCell>
-			<TableBodyCell>INA</TableBodyCell>
-			<TableBodyCell>$4450</TableBodyCell>
-			<TableBodyCell>5.68</TableBodyCell>
-		</TableBodyRow>
-		<TableBodyRow class="border-t-[1px]">
-			<TableBodyCell>2</TableBodyCell>
-			<TableBodyCell>Veddriq LEONARDO</TableBodyCell>
-			<TableBodyCell>INA</TableBodyCell>
-			<TableBodyCell>$4450</TableBodyCell>
-			<TableBodyCell>5.68</TableBodyCell>
-		</TableBodyRow>
-		<TableBodyRow class="border-t-[1px]">
-			<TableBodyCell>3</TableBodyCell>
-			<TableBodyCell>Veddriq LEONARDO</TableBodyCell>
-			<TableBodyCell>INA</TableBodyCell>
-			<TableBodyCell>$4450</TableBodyCell>
-			<TableBodyCell>5.68</TableBodyCell>
-		</TableBodyRow>
+		{#if male}
+			{#each worldRankingsMale as ranking, index (index)}
```

Comment:
```suggestion
			{#each (male ? worldRankingsMale:woldRankingsFemale) as ranking, index (index)}
```

Please do this like this (or similar) to avoid the code duplication.

Reply:
Oh yes I really like this solution! This makes it way cleaner